mlir/lib/Dialect/MIOpen/Generator/Conv2dGenerator.cpp

```diff
@@ -606,6 +606,12 @@ LogicalResult Conv2dGenerator::genConvModule(ModuleOp &module, int kernel_id,
         builder.getNamedAttr("xdlopsV2", builder.getBoolAttr(true)));
   }
 
+  // perf_config
+  if (!config.perfConfig.empty()) {
+    attributes.push_back(builder.getNamedAttr(
+        "perf_config", builder.getStringAttr(config.perfConfig)));
+  }
+
   // ignore tuning.
   if (ignoreTuning) {
     attributes.push_back(
```

Comment:
Could you help get rid of this `ignore_tuning` attribute. Now that you made `perfConfig` to be a independent attribute, this `ignore_tuning` doesn't seem necessary anymore.

Instead, you can make it much cleaner by such that: when `ignoreTuning == true`, assign an empty string to the perfConfig associated with the conv2d op.